advanced/Scripts/webpage.sh

```diff
@@ -620,6 +626,7 @@ main() {
         "resolve"             ) ResolutionSettings;;
         "addstaticdhcp"       ) AddDHCPStaticAddress;;
         "removestaticdhcp"    ) RemoveDHCPStaticAddress;;
+		"removedhcplease"	  ) RemoveDHCPLeaseAddress;;
```

Comment:
Minor change, but please make the indentation the same as the above and below lines :) (tabs -> spaces as below)

```suggestion
        "removedhcplease"     ) RemoveDHCPLeaseAddress;;
```